src/pages/[platform]/build-a-backend/add-aws-services/predictions/identify-text/index.mdx

```diff
@@ -0,0 +1,175 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Identify text',
+  description: 'Learn how to identify text from images and documents in your application using AWS Amplify.',
+  platforms: [
+    'javascript',
+    'angular',
+    'nextjs',
+    'react',
+    'vue'
+
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
```

Comment:
Can you add a banner for each of the pages telling customers that they are required to complete the getting started section first (where you set up the IAM roles with the right policy actions)

Reply:
Done